docs/observability/metrics.md

```diff
@@ -216,6 +216,9 @@ Number of current running migrations. Type: Gauge.
 ### kubevirt_vmi_migrations_in_scheduling_phase
 Number of current scheduling migrations. Type: Gauge.
 
+### kubevirt_vmi_migrations_in_undefined_phase
```

Comment:
nit: I like much more the "unset" wording from the phases `k6tv1.MigrationPhaseUnset` than undefined
which actually have pretty different meanings also

Reply:
I agree with @machadovilaca

Reply:
thanks, fixed (: